src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java

```diff
@@ -1965,6 +1967,19 @@ public IValue evaluate(OperatorBase.SafeVariablesGetter variables) throws Evalua
                     itemStack -> !itemStack.isEmpty() && itemStack.hasTag()
             )).build());
 
+    /**
+     * Get the tooltip lines of an itemstack.
+     */
+    public static final IOperator OBJECT_ITEMSTACK_TOOLTIP_LINES = REGISTRY.register(OperatorBuilders.ITEMSTACK_1_SUFFIX_LONG
+            .output(ValueTypes.LIST).symbol("tooltip_lines").operatorName("tooltiplines").interactName("tooltipLines")
+            .function(input -> {
+                ValueObjectTypeItemStack.ValueItemStack itemStack = input.getValue(0, ValueTypes.OBJECT_ITEMSTACK);
+                return ValueTypeList.ValueList.ofList(ValueTypes.STRING,
+                        itemStack.getRawValue().getTooltipLines(Minecraft.getInstance().player, TooltipFlag.Default.NORMAL).stream()
```

Comment:
`Minecraft` is a client-side-only class, so this will unfortunately crash when used on servers.
But I believe the first param of `getTooltipLines` is `Nullable`, so we can just pass `null` here.